src/javascript/app_2/pages/settings/sections/account_password.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import ContentHeader from '../components/content_header.jsx';
+
+class AccountPassword extends PureComponent {
+    render() {
+        const { title, description } = this.props;
+        return (
+            <div className='settings__content_container'>
+                <ContentHeader title={title} description={description} />
+                <div className='settings__content_form_container'>
+                    {/* content here */}
+                </div>
+            </div>
```

Comment:
All these lines define the layout of all settings components.
How about moving them to a single component? This will prevent repeating the same thing in many places, hence provides consistency and makes them easier to maintain in the future.